packages/core/integration-tests/test/html.js

```diff
@@ -146,73 +136,53 @@ describe('html', function() {
       }
     );
 
-    await assertBundleTree(b, {
-      name: 'index.html',
-      assets: ['index.html'],
-      childBundles: [
-        {
-          type: 'css',
-          assets: ['index.css'],
-          childBundles: [
-            {
-              type: 'js',
-              assets: [
-                'index.css',
-                'bundle-url.js',
-                'css-loader.js',
-                'hmr-runtime.js'
-              ],
-              childBundles: [
-                {
-                  type: 'map'
-                }
-              ]
-            },
-            {
-              type: 'map'
-            }
-          ]
-        }
-      ]
-    });
+    await assertBundles(b, [
+      {
+        name: 'index.html',
+        assets: ['index.html']
+      },
+      {
+        type: 'css',
+        assets: ['index.css']
+      },
+      {
+        type: 'js',
```

Comment:
This test doesn't quite work yet since we don't insert an HMR runtime into the page if there is no JS bundle. Before, we'd insert a JS bundle to handle updating the CSS when it changed. Might need to revisit that separately.